### DISCUSSION TITLE
New RSS Feed provider?

### DISCUSSION THREAD

**Discussion by fireshaper** [pwillworth/galaxyharvester#43] [Ideas]
Feedburner is no longer a thing so all the RSS feeds are broken now. Are there plans to change to something else?

**Comment by pwillworth** [comment]
Uh oh!  No, did not have a plan for that yet.  You know any good alternatives?

**Comment by fireshaper** [comment]
Maybe check out [follow.it](https://follow.it/intro?feedburner). They are one of the bigger alternatives.

**Comment by fireshaper** [comment]
Have there been any updates on this?

**Reply by pwillworth** [reply]
When I took a look at it, it seems the Feedburner is still working and the RSS feeds are still usable.  So, not a big priority at the moment.  Is something specific not working for you?

**Comment by fireshaper** [comment]
I tried accessing the feedburner links in a browser and the page just spins and doesn't load. But I looked at the source of the project and found a way to get the specific resources types from the main RSS feed. You can close this out.
